src/scripts/templates/gift_card.js

```diff
@@ -4,12 +4,10 @@
  * A file that contains scripts highly couple code to the Gift Card template.
  */
 
-import $ from 'jquery';
-
 const config = {
   qrCode: '#QrCode',
-  printButton: '#PrintGiftCard',
-  giftCardCode: '.giftcard__code',
+  printButton: 'PrintGiftCard',
+  giftCardCode: 'GiftCardDigits',
 };
 
 // new QRCode($qrCode[0], {
```

Comment:
This shouldn't be commented out. I would look at Narrative for how it is used. @martinamarien recently worked on that and might be able to answer any questions you have?

Reply:
Curious what the functionality of the QR code is supposed to be as well